### ISSUE TITLE
Consider setting mac settings supports-32-bit-f-cnt to true on TTN

### ISSUE THREAD

**Issue by MedadRufus** [ImperialSpaceSociety/LoRaMac-node#82]
By default, it counts to only 16 bit. While resets of fcount is enabled, if its disabled, the will run out after 65536. 

More information is here: https://www.thethingsindustries.com/docs/reference/cli/end_devices/#command-ttn-lw-cli-end-devices-create

The setting is here: 
```
--mac-settings.supports-32-bit-f-cnt

    type: bool

    default: false

    select the mac_settings.supports_32_bit_f_cnt field

```

**Comment by MedadRufus** [comment]
Use this reference: https://www.thethingsnetwork.org/forum/t/disable-frame-counter-checks-in-v3/35758/65

**Comment by MedadRufus** [comment]
```
ttn-lw-cli dev create **Your-App-ID** **Desired-Device-ID**
  --dev-eui **An-EUI**
  --frequency-plan-id EU_863_870_TTN
  --lorawan-version 1.0.3
  --lorawan-phy-version 1.0.3-a
  --abp  --session.dev-addr **260B-Not-Sure-Find-One**
  --session.keys.app-s-key.key **Random-Key**
  --session.keys.f_nwk_s_int_key.key **Another-Random-Key**
  --mac-settings.factory-preset-frequencies 867100000,867300000,867500000,867700000,867900000,868100000,868300000,868500000
  --mac-settings.rx1-delay RX_DELAY_5 
  --mac-settings.supports-32-bit-f-cnt 
  --mac-settings.resets-f-cnt
```

**Comment by MedadRufus** [comment]
fixed in #199
